src/Stratis.Bitcoin.Features.Miner/BlockDefinition.cs

```diff
@@ -10,12 +10,13 @@
 using Stratis.Bitcoin.Features.Consensus.Rules.CommonRules;
 using Stratis.Bitcoin.Features.MemoryPool;
 using Stratis.Bitcoin.Features.MemoryPool.Interfaces;
+using Stratis.Bitcoin.Mining;
 using Stratis.Bitcoin.Utilities;
-using static Stratis.Bitcoin.Features.Miner.PowBlockAssembler;
+using static Stratis.Bitcoin.Features.Miner.PowBlockDefinition;
 
 namespace Stratis.Bitcoin.Features.Miner
 {
-    public abstract class BlockAssembler
+    public abstract class BlockDefinition
```

Comment:
This is renamed to `BlockDefinition` I think we should provide an explanation and purpose at this level.
`A high level class that will allow the ability to override or inject functionality based on what type of block creation logic is used.`